components/combobox/private/menu.jsx

```diff
@@ -451,7 +455,7 @@ const Menu = (props) => {
 			item = React.cloneElement(
 				props.tooltipMenuItemDisabled,
 				tooltipProps,
-				menuItem[props.variant]
+				React.cloneElement(menuItem[props.variant], { tabIndex: 0 })
```

Comment:
`tabIndex: 0` should not be in menu items

Reply:
Related to my comment above on accessible tooltips.